packages/react/service_worker/OidcServiceWorker.ts

```diff
@@ -274,11 +279,14 @@ const handleMessage = (event: ExtendableMessageEvent) => {
       nonce: null,
       status: null,
       configurationName,
+      hideAccessToken: trustedDomainsHideAccessToken[configurationName],
```

Comment:
@krzempekk we need to evaluate this.

@guillaume-chervet we need to check if the trusted domains are something we can dynamically generate based on our deployment target. In our deployment model, we follow the: build once and deploy to many environments (dev/staging/prod). Therefore, anything hardcoded at the build time is a problem.

Reply:
Thank you for your feedback @darkowic.

I'am waiting for your response :)

Reply:
How can we set the access token to be shown?